skimage/measure/tests/test_regionprops.py

```diff
@@ -575,6 +577,17 @@ def test_regionprops_table():
                    'bbox+2': array([10]), 'bbox+3': array([18])}
 
 
+def test_regionprops_table_equal_to_original():
+    regions = regionprops(SAMPLE, INTENSITY_FLOAT_SAMPLE)
+    out_table = regionprops_table(SAMPLE, INTENSITY_FLOAT_SAMPLE,
+                                  properties=COL_DTYPES.keys())
+
+    for prop in COL_DTYPES.keys():
+        for i in range(len(regions)):
+            if not isinstance(regions[i][prop], Iterable):
+                assert regions[i][prop] == out_table[prop][i]
```

Comment:
Can't we test all the properties here?
```suggestion
        for i, reg in enumerate(regions):
            assert np.array_equal(reg[prop], out_table[prop][i])
```

Reply:
Yeah one can, but we need some trick since iterable properties such as `regions[i]["bbox"]` are stored as `out_table["bbox-0"][i]`, `out_table["bbox-1"][i]` etc., for example. (`_props_to_dict` does the job, but I think I shouldn't use it in the test itself as it also casts numbers). I think I was a bit lazy to skip iterable properties, and I will also add some tests for iterable cases and make a new PR!

Reply:
I updated the code to include iterable properties( pull request #5245 ). It's a little bit dirty hack but hopefully it suffices!